lib/connections/quic_listen.go

```diff
@@ -97,15 +97,19 @@ func (t *quicListener) serve(ctx context.Context) error {
 	}
 	defer func() { _ = udpConn.Close() }()
 
-	svc, conn := stun.New(t.cfg, t, udpConn)
-	defer conn.Close()
+	tracer := &writeTrackingTracer{}
+	quicTransport := &quic.Transport{
+		Conn:   udpConn,
+		Tracer: tracer,
+	}
```

Comment:
Should we `quicTransport.Close()` along size the `udpConn.Close()` ?

Reply:
Looks like it may not do much in the case where it didn't create the conn itself, but it also won't hurt so, yes, sure.